### PR TITLE
Fixes manual valve sprites and n2o scrubbing AND ventcrawling too now

### LINKED ISSUE #10674: Ventcrawler vision isn't working
scrnshot1
version: https://github.com/tgstation/-tg-station/commit/b18ea0296d5fdc87f90c9e109b05ae1b9b39bd12
i think, it's caused by #10560

### LINKED ISSUE #10683: Air alarm scrubbers button for N2o doesn't work
Clicking on the nitrous oxide scrubber button doesn't make the button light up or anything.

### PR DESCRIPTION
**Pull request by duncathan**
Both of these were me being shitty at cleanup
Fixes #10683 

sorry for blaming you mso

e: hey I fixed ventcrawling too; big thanks to @LaharlMontogmmery
Fixes #10674

### CODE REVIEW

File: code/modules/mob/living/ventcrawling.dm [hunk 1]
```diff
@@ -63,21 +63,21 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/components/unary
...
 
 	for(var/obj/machinery/atmospherics/A in totalMembers)
 		if(!A.pipe_vision_img)
 			A.pipe_vision_img = image(A, A.loc, layer = 20, dir = A.dir)
 			//20 for being above darkness
-		pipes_shown |= A.pipe_vision_img
+		pipes_shown += A.pipe_vision_img
 		if(client)
-			client.images |= A.pipe_vision_img
+			client.images += A.pipe_vision_img
```
Comment (phil235) [inline code/modules/mob/living/ventcrawling.dm]: why this change?

Comment (duncathan) [inline code/modules/mob/living/ventcrawling.dm]: discussion on #coderbus indicated that |= is slower than +=; because there's no way for there to be duplicate entries (dismissing varedit nonsense) |= is unnecessary

File: code/ATMOSPHERICS/datum_pipeline.dm [hunk 2]
```diff
@@ -121,11 +121,7 @@ var/pipenetwarnings = 10
 
 /obj/machinery/atmospherics/components/addMember(obj/machinery/atmospherics/A)
 	var/datum/pipeline/P = returnPipenet(A)
-	for(var/I = 1; I <= parents.len; I++)
-		if(parents["p[I]"] == P)
-			P.addMember(A, src)
-			break
-
+	P.addMember(A, src)
```
Comment (phil235) [inline code/ATMOSPHERICS/datum_pipeline.dm]: Is there any chance that P could be null?

Comment (duncathan) [inline code/ATMOSPHERICS/datum_pipeline.dm]: I don't actually know why I changed that in the first place. This is just restoring it to how it was before my refactor. It works fine.
https://github.com/tgstation/-tg-station/pull/10560/files#diff-5ab0e677d41168f5f7061e5ec68ab6f9L124
